project_and_code_guidelines.md

```diff
@@ -404,48 +401,102 @@ static final String ACTION_OPEN_USER = "com.myapp.action.ACTION_OPEN_USER";
 
 ### 2.2.14 Arguments in Fragments and Activities
 
-When data is passed into an Activity or Fragment via `Intents` or a `Bundles`, the keys for the different values __must__ follow the rules described in the section above. In addition, the constant should be defined in the `Activity` or `Fragment` class as `public`. 
+When data is passed into an `Activity `or `Fragment` via `Intents` or a `Bundles`, the keys for the different values __must__ follow the rules described in the section above.
 
-It's also a good practise to add a comment on top of the class to explain what values are expected and whether they are optional or required. 
+When an `Activity` or `Fragment` expect arguments, it should provide a `static public` method that facilitates the creation of the `Fragment` or `Intent`.
```

Comment:
so they're only used in the get start intent method and the unbundling after/during onCreate